### ISSUE TITLE
smmalloc

### ISSUE THREAD

**Issue by denji** [mjansson/rpmalloc-benchmark#3]
* https://github.com/SergeyMakeev/smmalloc

**Comment by Dan-Do** [comment]
https://github.com/microsoft/snmalloc
I see that @mjansson commited the folder, but don't see any updated result.

**Comment by mjansson** [comment]
Yeah, been very limited on personal project time lately and spending it on other things at the moment, haven't had time to generate updated results and graphs.
